attributes/network/11_ledger_tokens.adoc

```diff
@@ -0,0 +1,93 @@
+= Ledger Tokens Management Attribute (#11)
+:cddl: ./cddl/
+
+This attribute specifies that a ledger can have tokens managed by an authority.
+Without this attribute, tokens are defined at the genesis of the ledger, and cannot be created, minted, or burnt.
+
+Tokens described by this attribute are fungible and well specified.
+They would be equivalent to ERC-20 tokens.
+
+A server implementing this attribute SHOULD protect the endpoints described in this form in some way.
+For example, endpoints SHOULD error if the sender isn't from a certain address.
+
+== Creating Tokens
+
+One can create new tokens using the `tokens.create` command.
+
+[sources,cddl]
+----
+include::{cddl}/11_ledger_tokens.cddl[tag=create]
+----
+
+One can update the token information using the `tokens.update` command.
```

Comment:
Why can update a token? What is the permission scheme?

Reply:
Up to the server.